examples/generic_republish.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "07297a3d-048b-496b-adb0-8fdd67316021",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Hosts: Generic republishing\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60bd9de5-64c3-4f15-8824-ff1d94a894d7",
+   "metadata": {
+    "editable": true,
+    "jp-MarkdownHeadingCollapsed": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Setup\n",
+    "\n",
+    "See \"Hello OpenAssetIO\" notebook for details on how to bootstrap OpenAssetIO. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "df703c44ad53b21c",
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-03-13T08:15:00.152634Z",
+     "start_time": "2024-03-13T08:15:00.141508Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from resources import helpers\n",
+    "\n",
+    "manager, context = helpers.bootstrap(\"resources/generic_republish/openassetio_config.toml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a80594a-b171-49c5-8cfd-00e334759b1e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Getting started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In the following example we're going to ask the manager about the traits it supports with respect to an asset, and use that to determine a publishing workflow, without knowing anything about the entity.\n",
```

Comment:
[Consider] A sentence giving a contrived example as to why you might want to do this would aid in comprehensibility. You should always setup what the reader is about to digest up front so they can make a decision to continue reading or not ... at least that's what I think I'm no technical writer.

Maybe just :
> This notebook presents cases where the manager would need to agnostically republish entity data, such as republishing data mutated by a user, or stripping assets of non-required traits.

Reply:
Added an intro sentence at the top in d3c33698

I think our other notebooks could do with that too, but we established bad precedent with previous notebooks.